lib/attributor/types/hash.rb

```diff
@@ -19,24 +22,15 @@ class << self
     @key_attribute = Attribute.new(@key_type)
     @value_attribute = Attribute.new(@value_type)
 
-    def self.key_type=(key_type)
-      resolved_key_type = Attributor.resolve_type(key_type)
-      unless resolved_key_type.ancestors.include?(Attributor::Type)
```

Comment:
Because it'll never be hit, Attributor.resolve_type can never return a type that would not be valid. It'd raise an exception itself instead.